### ISSUE TITLE
debug command causes null pointer

### ISSUE THREAD

**Issue by kain184** [lucko/LuckPerms#897]
sponge version spongeforge-1.12.2-2611-7.1.0-BETA-2996.jar
server latest 12.2
luckperm version LuckPerms-Sponge-4.1.7.jar
```
[LP] Generating debugging output...
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: java.lang.NullPointerException
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at com.google.gson.JsonPrimitive.isPrimitiveOrString(JsonPrimitive.java:278)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at com.google.gson.JsonPrimitive.setValue(JsonPrimitive.java:101)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at com.google.gson.JsonPrimitive.<init>(JsonPrimitive.java:65)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.utils.gson.JObject.add(JObject.java:49)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.commands.impl.misc.DebugCommand.serializeMetaData(DebugCommand.java:247)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.commands.impl.misc.DebugCommand.lambda$getPlayersData$6(DebugCommand.java:204)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.utils.gson.JObject.add(JObject.java:65)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.commands.impl.misc.DebugCommand.getPlayersData(DebugCommand.java:197)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.commands.impl.misc.DebugCommand.execute(DebugCommand.java:94)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.commands.abstraction.SingleCommand.execute(SingleCommand.java:51)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.commands.abstraction.SingleCommand.execute(SingleCommand.java:43)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.commands.CommandManager.execute(CommandManager.java:201)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at me.lucko.luckperms.common.commands.CommandManager.lambda$onCommand$0(CommandManager.java:146)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
04.04 17:07:23 [Server] pool-5-thread-1/INFO [STDERR]: [me.lucko.luckperms.common.commands.CommandManager:execute:205]: 	at java.lang.Thread.run(Thread.java:748)
```

**Comment by lucko** [comment]
Afaik this is already fixed in the latest version.

Can you try updating?
https://luckperms.github.io/

**Comment by lucko** [comment]
Yeah, fixed in 1689a8abe28d5361f50eedddead567d334041e07
